userspace/libsinsp/eventformatter.cpp

```diff
@@ -163,6 +163,42 @@ bool sinsp_evt_formatter::on_capture_end(OUT string* res)
 	return res->size() > 0;
 }
 
+bool sinsp_evt_formatter::resolve_tokens(sinsp_evt *evt, map<string,string>& values)
+{
+	bool retval = true;
+	const filtercheck_field_info* fi;
+
+	ASSERT(m_tokenlens.size() == m_tokens.size());
+
+	uint32_t j = 0;
+	for(j = 0; j < m_tokens.size(); j++)
+	{
+		char* str = m_tokens[j]->tostring(evt);
+
+		if(str == NULL)
+		{
+			if(m_require_all_values)
+			{
+				retval = false;
```

Comment:
I'm not sure I understand this correctly 100%, but if we require_all_values, wouldn't it be enough to `break` immediately instead of keep looking at following tokens?